compaction_test.go

```diff
@@ -249,22 +285,65 @@ func TestCompaction(t *testing.T) {
 					[]expectedPart{
 						{
 							numRowGroups:    1,
-							compactionLevel: compactionLevel1,
+							compactionLevel: parts.CompactionLevel1,
 							data:            []int64{9},
 						},
 					},
 				},
 			},
 		},
+		{
+			name: "ArrrowSimpleLevel0ToLevel1",
+			// Insert three rows.
+			inserts: []int64{1, 2, 3, compactCommand},
+			// Insert as an arrow record
+			record: true,
```

Comment:
We could also rerun all the test cases above but with arrow records as input instead of parquet, what do you think?

Reply:
That's a great idea